docs/src/createEmotionCache.ts

```diff
@@ -1,6 +1,21 @@
+/* eslint-disable default-case */
 import createCache from '@emotion/cache';
+import { Element, RULESET } from 'stylis';
+
+function globalSelector(element: Element) {
```

Comment:
A comment to explain the transformation done could be great.

Reply:
Will do!